concrete/src/Multilingual/Service/Extractor.php

```diff
@@ -69,7 +69,7 @@ public function extractTranslatableSiteStrings()
             foreach ($packages as $package) {
                 $fullDirname = DIR_PACKAGES.'/'.$package->getPackageHandle();
                 $phpParser->parseDirectory($fullDirname,
-                    DIRNAME_PACKAGES.'/'.$dirname,
```

Comment:
What about reusing `$fullDirname`?

Reply:
Because it's better to see the relative path instead of the absolute path (it's shorter and portable)